addons/account/models/account_bank_statement.py

```diff
@@ -35,7 +35,7 @@ class AccountBankStatement(models.Model):
     # keeping this order is important because the validity of the statements are based on their order
     first_line_index = fields.Char(
         comodel_name='account.bank.statement.line',
-        compute='_compute_date_index', store=True, index=True,
+        compute='_compute_date_index', store=True,
```

Comment:
probably better to set `index=False` explicitly? Are all queries that searches on this field always searched with a `journal_id`, because else the compound index will not be hit?

Reply:
The index only makes sense withing a journal indeed, so we should always hit the compound index